### PR TITLE
Create README for SYS-2.1: Ingress control-plane ACL.

### PR DESCRIPTION
**Pull request by sugrimov**

### CODE REVIEW

File: feature/system/control-plane-traffic/otg_tests/ingress_acl/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    - Allow gRPC from any (lab management access)
+    - Allow SSH from MGMT-SRC
+    - Allow ICMP from MGMT-SRC
+    - Explicit deny
+
+3.  Apply filter to control-plane ingress.
+
+## Test cases
+
+### SYS-2.1: Verify ingress control-plane ACL permit
```
Comment (dplore) [inline feature/system/control-plane-traffic/otg_tests/ingress_acl/README.md:28]:
```suggestion
### SYS-2.1.1: Verify ingress control-plane ACL permit
```

File: feature/system/control-plane-traffic/otg_tests/ingress_acl/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+Generate ICMP traffic to device loopback from MGMT-SRC
+Generate SSH SYN packets to device loopback from MGMT-SRC
+
+Verify:
+
+*  ACL counters for corresponding ACL entries are incrementing.
+*  Device responds to ICMP permitted
+*  Device sends TCP-ACK for SSH session
+
+### SYS-2.2: Verify control-plane ACL deny
```
Comment (dplore) [inline feature/system/control-plane-traffic/otg_tests/ingress_acl/README.md:38]:
```suggestion
### SYS-2.1.2: Verify control-plane ACL deny
```

File: feature/system/control-plane-traffic/otg_tests/ingress_acl/README.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
+# SYS-2.1: Ingress control-plane ACL.
+
+## Summary
+
+The test verifies securing device control-plane access with an ingress access-control-list (ACL).
+
+## Setup
+
```
Comment (dplore) [inline feature/system/control-plane-traffic/otg_tests/ingress_acl/README.md:8]: Specify the topology, choosing from currently defined topologies.  See example in template at: https://github.com/openconfig/featureprofiles/blob/main/doc/test-requirements-template.md#testbed-type

**Review by dplore — COMMENTED** [review]
Looks good, just a couple small comments

**Review by ihebboubaker — APPROVED** [review]

**Review by dplore — APPROVED** [review]

**Pull request merged**